src/jit.rs

```diff
@@ -69,7 +69,7 @@ const TARGET_OFFSET: isize = ebpf::PROG_MAX_INSNS as isize;
 const TARGET_PC_EXIT: isize = TARGET_OFFSET + 1;
 const TARGET_PC_EPILOGUE: isize = TARGET_OFFSET + 2;
 const TARGET_PC_DIV_BY_ZERO: isize = TARGET_OFFSET + 3;
-const TARGET_NO_ACCESS_VIOLATION: isize = TARGET_OFFSET + 4;
+const TARGET_PC_EXCEPTION_AT: isize = TARGET_OFFSET + 4;
```

Comment:
Rebase would remove these changes from other prs?

Reply:
Yes, they should not appear in the diff anymore.